ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.18__periodics.yaml

```diff
@@ -103,6 +108,17 @@ tests:
   steps:
     cluster_profile: hypershift
     workflow: hypershift-aws-conformance-proxy
+- as: e2e-aws-n-1-upgrade
```

Comment:
Can you remove the `n-1` from the name?  `upgrade` is sufficient and, IIUC, implies a y-stream upgrade job for TRT/Sippy